### DISCUSSION TITLE
BigWig track set to bin size of 50?

### DISCUSSION THREAD

**Discussion by azatian** [GMOD/jbrowse-components#2693] [Q&A] [answered]
Hello! So the bigwig tracks that I have uploaded onto our JBrowse 2 is looking less like a signal (see the BAM underneath) and more like a bar chart/histogram. Is there a way to not set the bin size? It seems like the bigwig is displaying values grouped on bins of equal length and 50bp. We did not have this issue in JBrowse 1, so I'm wondering if there is something inadvertent happening with the way we are displaying the track or perhaps it's an issue with my bigwig file. Thank you for your help
![Screen Shot 2022-01-28 at 4 30 24 PM](https://user-images.githubusercontent.com/9220290/151630906-6d38cf60-fb8d-4a84-a3dc-8792e4c3bbd5.png)

**Comment by cmdcolin** [comment, marked as answer]
this might be due to how you created your bigwig file. jbrowse will display per-base data from bigwig files but some tools automatically create a bin size. for example https://deeptools.readthedocs.io/en/develop/content/tools/bamCoverage.html has a default --binSize of 50 (you can say --binSize 1 to modify it). also i did a little dive into coverage tools here, kind of technical just to see how different tools generate possibly different values https://gist.github.com/cmdcolin/9f677eca28448d8a7c5d6e9917fc56af

**Reply by azatian** [reply]
Ahh, you are so right! We made the bigwig with default settings. I will go ahead and change it to a bin size of 1. Nice comparison of the tools, we are using deeptools. Thank you so much!!
